app/services/signature-service.js

```diff
@@ -38,4 +38,20 @@ export default class SignatureService extends Service {
       await signMarkingActivity.save();
     }
   }
+
+  async unmarkDocumentForSignature(piece) {
+    const signMarkingActivity = await piece.signMarkingActivity;
+    const signPreparationActivity = await signMarkingActivity.signPreparationActivity;
+
+    if (signPreparationActivity) {
```

Comment:
Niet dat er al voorbereidingsactiviteiten zijn om op te kunnen stoppen, maar ik zou toch die toaster al verder uitwerken dat we in de toekomst dan wel weten waarom die niet ongemarkeerd worden als we er zouden op klikken (als er een voorbereidingsactiviteit op hangt)

Reply:
ik heb het toegevoegd, en even preparation data aangemaakt om te testen

De alert-stack toont wel niet door css issue?
Even lokaal css aangepast om toch te kunnen tonen.
@brenner-company ingelicht

![image](https://user-images.githubusercontent.com/22245223/135625046-698e0297-fc9f-4928-a1cd-555334e2a210.png)

Reply:
Tekst is later nog aanpasbaar, zou dat pas doen als de gebruiker het zelf kunnen triggeren, wat nu nog onmogelijk is.

Reply:
@ValenberghsSven @TaneeG Issue ivm fullscreen modal & alerts wordt opgenomen onder https://kanselarij.atlassian.net/browse/KAS-2944.